os/fs/driver/mtd/smart.c

```diff
@@ -19,6 +19,8 @@
  * fs/driver/mtd/smart.c
  *
  * Sector Mapped Allocation for Really Tiny (SMART) Flash block driver.
+ * journaling mechanism is implemented at this layer itself since 2020, 
```

Comment:
Let's update to "Journaling mechanism is added to the SMART layer itself and upgrades smartfs to jSmartFs"